Crash when opening downloaded podcasts

Steps to reproduce:
- Open "Downloaded Podcasts" from the side menu
- App crashes

Specifications:
- Android 13 / Api 33
- `devDebug` variant built from master ab32d65b56edac251befb1bb4bb182e77f88047e

Stacktrace:
```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:381)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:250)
	at java.util.concurrent.FutureTask.run(FutureTask.java:269)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.NoSuchMethodError: No interface method toList()Ljava/util/List; in class Ljava/util/stream/Stream; or its super classes (declaration of 'java.util.stream.Stream' appears in /apex/com.android.art/javalib/core-oj.jar)
	at de.luhmer.owncloudnewsreader.NewsReaderDetailFragment$UpdateCurrentRssViewTask.doInBackground(NewsReaderDetailFragment.java:542)
	at de.luhmer.owncloudnewsreader.NewsReaderDetailFragment$UpdateCurrentRssViewTask.doInBackground(NewsReaderDetailFragment.java:497)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137) 
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637) 
	at java.lang.Thread.run(Thread.java:1012) 
```

--- a/News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderDetailFragment.java
+++ b/News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderDetailFragment.java
@@ -539,7 +539,7 @@ public class NewsReaderDetailFragment extends Fragment {
                 items = items.stream().filter((rss) -> {
                     var podcast = DatabaseConnectionOrm.ParsePodcastItemFromRssItem(mActivity, rss);
                     return podcast.offlineCached;
-                }).toList();
+                }).collect(Collectors.toList());
             }
 
             sw.stop();

